tests/cases/php/separate-use

```diff
@@ -0,0 +1,24 @@
+---DESCRIPTION---
+One use statement per declaration
+---CONTENTS---
+<?php
+
+namespace Vendor;
+
+use Foo, Bar;
```

Comment:
No `function` and `const` examples yet, waiting for https://github.com/squizlabs/PHP_CodeSniffer/pull/2143 to be released.

Reply:
How does this deal with PHP 7s grouped statements 
```php
use some\namespace\{ClassA, ClassB, ClassC as C};
```

Reply:
5941c0cb6d58a678f09fe8f4a6ebe39740b4b413

Reply:
ah nice it does it for the functions and consts too 👍